Experimentation/AzureBlobDownloader.cs

```diff
@@ -252,8 +255,14 @@ await item.Blob.DownloadToFileAsync(temp, FileMode.Create, AccessCondition.Gener
                 {
                     var top = priorityQueue.First;
 
+                    var jObject = JObject.Parse(top.Line);
```

Comment:
maybe add a try catch, just in case the json is invalid so we don't loose the complete file.
not sure if skipping or injecting some infromation e.g. /* exception: ... */ is better